docs/tables/pagerduty_vendor.md

```diff
@@ -0,0 +1,70 @@
+# Table: pagerduty_vendor
+
+A PagerDuty vendor represents a specific type of integration, i.e. AWS Cloudwatch, Splunk, Datadog etc.
+
+**Note:**It is recommended that queries to this table should include (usually in the `where` clause) at least one of these columns: `name` or `id`.
+
+## Examples
+
+### List vendor for AWS CloudWatch integration
```

Comment:
Do we only get 1 vendor/row from this query, or are multiple returned?

Reply:
@cbruno10  This query will return 1 row that contains information about `AWS CloudWatch` integration.
```
+-------------------+---------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| name              | id      | description                                                                                                                                                                      |
+-------------------+---------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Amazon CloudWatch | PZQ6IND | Amazon Web Services CloudWatch provides monitoring for AWS cloud resources and customer-run applications.                                                                        |
|                   |         |                                                                                                                                                                                  |
|                   |         | This integration supports alarms from CloudSearch, DynamoDB, EBS, EC2, ECS, ElastiCache, ELB, ES, Kinesis, Lambda, ML, Redshift, RDS, Route53, SNS, SQS, S3, SWF, StorageGateway |
+-------------------+---------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```